### DISCUSSION TITLE
Disscution!

### DISCUSSION THREAD

**Discussion by ionuttbara** [ionuttbara/windows-defender-remover#12] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gamesturbator** [comment]
Hi, I'm clueless about navigating Github. Which link do I click to download the actual program? Thanks.

**Reply by ionuttbara** [reply]
going to release sections and see  the versions.

**Reply by gamesturbator** [reply]
Ha HA, I can't see or find the release section. I'm a bit thick.😁 Thanks. Edit: Found it. The main page I was sent to didn't show the release section. I clicked "go to file" instead of clicking the blue linked words
 "windows-defender-remover" that I was taken to the release section.

**Reply by ionuttbara** [reply]
Now you can download the script from Softpedia and instalki.pl , if you want.

**Reply by JunaidJamshad20** [reply]
> Now you can download the script from Softpedia and instalki.pl , if you want.

By opening Softpedia or instalki.pl, you would be able to download hundreds of programs. Business needs more business management alternatives raise business organizations for stock managing. It is giving its users Cloud Stock Management Software at low-cost prices and ensured resource dependency so that the effectiveness of its user in the stock inventory management by such software improves. It may actually be a matter of importance in checking the quantities by seeing your data regarding your Stock Information - How to get free software today via your use of free downloading of [Stock Manager Software](https://www.ictsystems.com.pk/webdesk-trading-software/) with help of our use of software with an assistance. Be sure to check the features of the site you may want to pick for your business, from reviews and overall suitability.

**Comment by katastrophe92** [comment]
@jbara2002 First of all thank you for this amazing repository, I better ask this before: all the changes done by your software is 100% reversible? Thanks

**Reply by ionuttbara** [reply]
yes and no.
If you disable Windows Defender using the script, is reversible and can be enabled with the same script.
If you "remove" the Defender, this change is not reversible. You must do an upgrade in-place if you want to restore remover changes.

**Reply by sigmaphi007** [reply]
@jbara2002 was about to ask the same question. I will be using something else as Antivirus! Normally would have left it but damn in Win 11 it as an annoying false positive & every time I allow it just kept deleting the application. Had enough & used your app. Much appreciated

**Comment by captainmaged** [comment]
Is nsudo needed or plain run as admin will be enough?

**Reply by ionuttbara** [reply]
just run as admin is sufficient.
No NSudo Needed

**Comment by justhelppk** [comment]
how to enable windows defender once I disabled it using this script. now I want to enable this.

**Comment by GODZILLA2169** [comment]
thank you so much windows defender kept removing modding files for games via thinking it was a trojan horse exploit and had to restore them everytime.
